modules/core/src/v2/coins/fiat.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @prettier
+ */
+import {
+  BaseCoin,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+} from '../baseCoin';
+import { MethodNotImplementedError } from '../../errors';
+
+export abstract class Fiat extends BaseCoin {
+  /**
+   * Returns the factor between the base unit and its smallest subdivison
+   * @return {number}
+   */
+  getBaseFactor() {
+    return 1e2;
+  }
+
+  getChain() {
+    return 'fiat';
+  }
+
+  getFamily() {
+    return 'fiat';
+  }
+
+  getFullName() {
+    return 'Fiat';
+  }
+
+  /**
+   * Return whether the given m of n wallet signers/ key amounts are valid for the coin
+   */
+  isValidMofNSetup({ m, n }: { m: number; n: number }) {
+    return m === 0 && n === 0;
+  }
+
+  isValidAddress(address: string): boolean {
+    throw new MethodNotImplementedError();
+  }
+
+  generateKeyPair(seed?: Buffer): KeyPair {
+    throw new MethodNotImplementedError();
+  }
+
+  isValidPub(pub: string): boolean {
+    throw new MethodNotImplementedError();
+  }
+
+  async parseTransaction(params: ParseTransactionOptions): Promise<ParsedTransaction> {
+    return {};
```

Comment:
please throw if you're not ready to implement these yet, or you can also leave them unimplemented in the abstract base class and throw in the derived class instead.
```suggestion
    throw new MethodNotImplementedError();
```